packages/eas-cli/src/commands/release/rename.ts

```diff
@@ -0,0 +1,140 @@
+import { getConfig } from '@expo/config';
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../graphql/client';
+import log from '../../log';
+import { ensureProjectExistsAsync } from '../../project/ensureProjectExists';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+
+type UpdateRelease = {
+  id: string;
+  releaseName: string;
+};
+
+async function renameUpdateReleaseOnAppAsync({
+  appId,
+  currentName,
+  newName,
+}: {
+  appId: string;
+  currentName: string;
+  newName: string;
+}): Promise<UpdateRelease> {
+  const data = await withErrorHandlingAsync(
+    graphqlClient
+      .mutation<
+        { updateRelease: { editUpdateRelease: UpdateRelease } },
+        {
+          input: {
+            appId: string;
+            releaseName: string;
+            newReleaseName: string;
+          };
```

Comment:
```suggestion
          input: EditUpdateReleaseInput;
```
(Need to run `yarn generate-graphql-code` so this gets added to `generated.ts`.)